source/process/handbook.rst

```diff
@@ -1,3 +1,35 @@
+Team Handbook
+#############
+
+Mission & Vision
+-------
+
+Our mission is: "to give every organization the power to achieve their highest priorities through modern communication."
+
+We intend to disrupt the global workplace communications market with open core solutions delivering irresistible value.
+
+Values
+--------
+
+Values are deliberate choices defining our behavior. When facing complexity, uncertainty, or ambiguity (CUA) we determine our point of view and our actions through the lens of our values: 
+
+Customer Obsession 
+==================
+We exist to serve customers. In everything we do, start with customer perspective and work backwards. Earn and keep their trust. 
```

Comment:
Not sure how I feel about the wording "We exist to serve customers", but maybe we can discuss and update it later.